src/recipients/RecipientItemV2.tsx

```diff
@@ -30,6 +30,7 @@ function RecipientItem({ recipient, onSelectRecipient, loading, selected }: Prop
   const { t } = useTranslation()
 
   const onPress = () => {
+    Keyboard.dismiss()
```

Comment:
this seems a bit out of place in the component. Is this something can be done in the onSelectRecipient callback itself?

Reply:
Open to moving it, but I also can't think of a case we wouldn't want to dismiss the keyboard on a recipient press. This way we don't have a the dismiss in both `setSelectedRecipientWrapper` and `onSelectRecentRecipient`.